index.bs

```diff
@@ -8021,6 +8056,22 @@ The <dfn>Credential Parameters</dfn> is a JSON [=Object=] passed to the [=remote
                 </td>
                 <td>string</td>
             </tr>
+            <tr>
+                <td>|userLanguage|</td>
+                <td>
+                    The [=resource-wide default=] language tag for the {{PublicKeyCredentialUserEntity|user}} entity associated to the credential.
+                    This property may not be defined.
```

Comment:
```suggestion
                    Note that this property might not be present.
```

Reply:
Thanks, but I'm going to reject this proposal as "This property may not be defined" is the formulation already used for `userHandle` and `largeBlob`:

https://github.com/w3c/webauthn/blob/6507c4c49cd8919d2745b35f016abe2729a79a72/index.bs#L8003-L8006

https://github.com/w3c/webauthn/blob/6507c4c49cd8919d2745b35f016abe2729a79a72/index.bs#L8016-L8019